db/migrate/20150818142457_add_vm_template_id_to_vms.rb

```diff
@@ -0,0 +1,5 @@
+class AddVmTemplateIdToVms < ActiveRecord::Migration
+  def change
+    add_column :vms, :vm_template_id, :bigint
```

Comment:
hm, so it's just not exposed in the UI? I could't find it anywhere. I'll check the code for genealogy.

btw. it was in the John's cross links requirements, so I assumed it's not there